### PR TITLE
Feature/flinkster cab rework

### PR DESCRIPTION
**Pull request by dennispost**

### CODE REVIEW

**Comment by dennispost** [comment]
Todos:

- [x] Create separate Area Model
- [x] Find solution for MultiPolygon Coordinates

**Comment by dennispost** [comment]
@lightsprint09 How can we use this https://github.com/ghengeveld/graphql-geojson to support GeoJSON polygons?

**Comment by lightsprint09** [comment]
Good question build our own type

**Comment by dennispost** [comment]
@lightsprint09 this module provides a MultiPolygonObject already (see https://github.com/ghengeveld/graphql-geojson-demo/blob/master/src/schema.js). Is there a way to use this in our 'GraphQL schema language' or do we need to rebuild our schema?

File: Flinkster/FlinksterBike.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
 
-
-const Location = require('./location');
-const MailAddress = require('./mailAddress');
+const Location = require('../location');
+const MailAddress = require('../mailAddress');
+const FlinksterParkingArea = require('../FlinksterParkingArea');
 
 class Attributes {
   constructor(att) {
```
Comment (lightsprint09) [inline Flinkster/FlinksterBike.js]: rename parameter `att ` to attributes

File: Flinkster/FlinksterCar.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+const access = require('safe-access');
+
+const Location = require('../location');
+const MailAddress = require('../mailAddress');
+const FlinksterParkingArea = require('../FlinksterParkingArea');
+
+class Attributes {
+  constructor(att) {
```
Comment (lightsprint09) [inline Flinkster/FlinksterCar.js]: rename parameter `att ` to attributes

File: Flinkster/FlinksterCar.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    this.grossamount = price.grossamount;
+    this.currency = price.currency;
+    this.taxrate = price.taxrate;
+    this.preferredprice = price.preferredprice;
+  }
+}
+
+class CarEquipment {
+  constructor(equipment) {
+    this.cdPlayer = access(equipment, 'cdPlayer');
```
Comment (lightsprint09) [inline Flinkster/FlinksterCar.js]: try `if (!equipment) {
return
}`

File: Flinkster/FlinksterLoader.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        if (result.items) {
+          return result.items;
+        }
+        return [];
+      });
+
+    return promise;
+  }
+
+  nearbyBikes(latitude, longitude, radius, count, offset) {
```
Comment (lightsprint09) [inline Flinkster/FlinksterLoader.js]: duplicated with nearbyCars

File: FlinksterParkingArea.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        features: [{
+          type: 'Feature',
+          properties: null,
+          geometry: {
+            type: 'MultiPolygon',
+            coordinates: parkingArea.geometry.position.coordinates,
+          },
+        }] };
+
+      for (const p of parkingArea.geometry.position.coordinates) {
```
Comment (lightsprint09) [inline FlinksterParkingArea.js]: remove

File: schema.js [hunk 6]
```diff
@@ -133,11 +133,11 @@ const schema = buildSchema(`
   }
 
   type Nearby {
-    stations: [Station]
-    parkingSpaces: [ParkingSpace]
+    stations (radius: Int = 10000, count: Int = 10, offset: Int = 0): [Station]!
```
Comment (lightsprint09) [inline schema.js]: Mark Elements inside of an array as non optional

**Review by lightsprint09 — CHANGES_REQUESTED** [review]

**Pull request merged**